app/controllers/pending_loans_controller.rb

```diff
@@ -0,0 +1,70 @@
+class PendingLoansController < ApplicationController
+  def create
+    update_cart
+    flash[:notice] = "Added to Pending Loans"
+    redirect_to pending_loan_path
+  end
+
+  def show
+    @pending_loans = {}
+    session[:pending_loan].each do |project_id, loan_amount|
+      @pending_loans[Project.find(project_id.to_i)] = loan_amount
+    end
+  end
+
+  def checkout_cart
+    if current_user.nil?
+      flash[:alert] = "You must login to checkout"
+      redirect_to showcart_path
+    else
+      redirect_to create_order_path
+    end
+  end
+
+  def delete_pending_loan
+    if params[:pending_loan].nil?
+      delete_all_projects_from_cart
+    else
+      delete_specific_project_from_cart
+      flash[:notice] = "Project removed from cart"
+    end
+    redirect_to pending_loan_path
+  end
+
+  def update_project_amount
+    session[:pending_loan][params[:update_pending_loan_amount][:project_id]] =
+       params[:update_pending_loan_amount][:loan_amount]
+    flash[:notice] = 'Project loan amount updated'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.